### DISCUSSION TITLE
How to insert a empty paragraph after a selected Node.

### DISCUSSION THREAD

**Discussion by 4-1-1** [Milkdown/milkdown#493] [Q&A]
I made a new block node. but it can not to blur by mouse click or keyboard.
So I need to insert a editable node like a empty paragraph

**Comment by 4-1-1** [comment]
Have done it 🤒

**Reply by Himokow** [reply]
Hello, could you share the answer ? I've got the same issues

**Reply by Saul-Mirone** [reply]
```typescript
const { doc, schema, tr } = state;
const type = schema.nodes['paragraph'];
const transaction = tr.insert(doc.content.size, type.create());
view.dispatch(transaction);
```

**Reply by 4-1-1** [reply]
@Himokow  code in here  https://github.com/Saul-Mirone/milkdown/pull/537
